src/webservice/Router.h

```diff
@@ -71,7 +72,7 @@ class Router final : public cpp::NonCopyable, public cpp::NonMovable {
 public:
     explicit Router(const std::string &prefix) : prefix_(prefix), webSvc_(nullptr) {}
     Router(const std::string &prefix, const WebService *webSvc) : prefix_(prefix), webSvc_(webSvc) {
-        DCHECK_NOTNULL(webSvc);
+        UNUSED(DCHECK_NOTNULL(webSvc));
```

Comment:
You can put this DCHECK in initializer list like: `webSvc_(DCHECK_NOTNULL(webSvc))` rather than using the `UNUSED`